docs/users/FAQ.md

```diff
@@ -0,0 +1,35 @@
+This page includes answers to the commonly asked questions about the app. 
+
+## Why doesn't the app support my game?
+The Nexus Mods app is still in development and will not support specific games because these features have not been included yet. Our primary focus is to establish core functionality and foundational elements. 
+
+Adding support for a wide range of games it not a practical decision at this stage of developement, as it can introduce additional complexity and obscure underlying issues that need to be resolved first. The goal is to ensure the mod manager works seamlessly for a select few games before expanding compatibility in subsequent beta and final releases.
+
+Our plan is to go game-by-game and work with the modding community at each step to ensure we're building something great. To see which games are supported, coming next and planned for the future please see the [roadmap](https://trello.com/b/gPzMuIr3/nexus-mods-app-roadmap).
```

Comment:
Note: The linked roadmap is not publicly visible

Reply:
Yep, I am aware of this. It's up to Luke to finish it up and publish it.

Reply:
I didn't even know we had one...